Backend/automatic testing

HTOdinTH opened the pull request:
Tests only run on pull requests, so I need this pr for testing. Not ready for merging yet

haakofli committed:
Merge pull request #39 from OpenDataNTNU2/dev

Dev

HTOdinTH committed:
Create dotnet-testing.yml

HTOdinTH committed:
specify dotnet directory for automatic testing

ingin97 reviewed (dismissed):
Du var visst kanskje ikke helt ferdig.

HTOdinTH committed:
Merge remote-tracking branch 'origin/dev' into backend/automatic_testing

HTOdinTH committed:
Fixed Coordination and Dataset so Publisher and Coordination are nullable

plain ints are not nullable, which causes some databases (sqlite, for testing) to treat them as required

HTOdinTH committed:
Changed DatasetServiceTests.TestUpdate to succeed despite missing functionality

This is to avoid testing output confusion. UpdateAsync is not a fully featured method and should not pass all tests

HTOdinTH committed:
Merge remote-tracking branch 'origin/dev' into backend/automatic_testing

HTOdinTH committed:
Rename tests.csproj to Tests.csproj

HTOdinTH committed:
Merge branch 'backend/automatic_testing' of https://github.com/OpenDataNTNU2/TDT4290-OpenData2020 into backend/automatic_testing

HTOdinTH committed:
Reduce test output verbosity

HTOdinTH committed:
Suppress warning for unimplemented test

HTOdinTH commented:
Up and running now! Ready for merging

heddal committed:
Merge branch 'dev' into backend/automatic_testing
